lib/inject.js

```diff
@@ -140,6 +138,25 @@ function injectPa11y(window, options, done) {
 		return false;
 	}
 
+	function hiddenAreasContainsElement(element, hiddenSelectors) {
+		var hiddenElements = window.document.querySelectorAll(hiddenSelectors);
+
+		for(var i = 0; i < hiddenElements.length; i++) {
+			if(element.isEqualNode(hiddenElements[i])) {
```

Comment:
I would have preferred to use `Element.matches` but unfortunately its not supported within Phantom and polyfilling it would have meant faking a lot of the DOM in the unit tests so I have decided to go with `isEqualNode` instead. 

This unfortunately this means I have had to use `querySelectorAll` and loop over every node it matches.